@xen-orchestra/web/src/pages/host/[id]/networks.vue

```diff
@@ -1,24 +1,49 @@
 <template>
   <div class="networks">
     <UiCard class="container">
-      <HostPifsTable :host />
+      <HostPifTable :pifs />
     </UiCard>
+    <HostPifSidePanel v-if="selectedPif" :pif="selectedPif" />
+    <UiPanel v-else>
+      <VtsNoSelectionHero type="panel" />
+    </UiPanel>
   </div>
 </template>
 
 <script setup lang="ts">
-import HostPifsTable from '@/components/host/HostPifsTable.vue'
-import type { XoHost } from '@/types/xo/host.type'
+import HostPifSidePanel from '@/components/host/HostPifSidePanel.vue'
+import HostPifTable from '@/components/host/HostPifTable.vue'
+import { usePifStore } from '@/stores/xo-rest-api/pif.store'
+import type { XoPif } from '@/types/xo/pif.type'
+import VtsNoSelectionHero from '@core/components/state-hero/VtsNoSelectionHero.vue'
 import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiPanel from '@core/components/ui/panel/UiPanel.vue'
+import { useRouteQuery } from '@core/composables/route-query.composable'
+import { computed } from 'vue'
+import { useRoute } from 'vue-router/auto'
 
-defineProps<{
-  host: XoHost
-}>()
+const { records } = usePifStore().subscribe()
+const route = useRoute()
```

Comment:
To fix `type-check` error l.30:
```suggestion
const route = useRoute<'/host/[id]'>()
```